packages/netlify-cms-core/src/components/MediaLibrary/MediaLibrary.js

```diff
@@ -34,7 +34,7 @@ class MediaLibrary extends React.Component {
     this.props.loadMedia();
   }
 
-  componentWillReceiveProps(nextProps) {
+  UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
@erquhart Can you explain why we just toggle a `visible` switch on the media library instead of unmounting/remounting like a regular component? Is it just a performance issue with all of the thumbnails?

Reply:
I believe that was it, don't want to keep reloading the thing.